hazelcast/src/main/java/com/hazelcast/map/impl/client/MapEntrySetRequest.java

```diff
@@ -53,15 +54,12 @@ protected OperationFactory createOperationFactory() {
 
     @Override
     protected Object reduce(Map<Integer, Object> map) {
-        MapEntrySet entrySet = new MapEntrySet();
+        Set<Map.Entry<Data, Data>> entrySet = new HashSet<Map.Entry<Data, Data>>();
```

Comment:
Please have a look at this change! It's done the same way in `MapKeySetRequest` and `MapValuesRequest`, so we can make use of the built-in `addAll()` method.